src/qiboconnection/api.py

```diff
@@ -367,8 +367,6 @@ def execute(
         if not selected_devices:
             raise ValueError("No devices were selected for execution.")
 
-        if not isinstance(circuit, list):
-            circuit = [circuit]
```

Comment:
Why are you removing this?

Reply:
as discussed in the daily, we'll need to do two releases due to how is out CI/CD designed. This will be added in the latter